.github/ISSUE_TEMPLATE.md

```diff
@@ -24,6 +24,10 @@
 <!--- How has this issue affected you? What are you trying to accomplish? -->
 <!--- Providing context helps us come up with a solution that is most useful in the real world -->
 
+## Regression
+<!-- Is this issue a regression? (Yes / No) -->
+<!-- If Yes, optionally please include minio version or commit id or PR# if you have these details. -->
```

Comment:
Can we clarify this a little further like, include minio version or commit id or PR# that caused this regression, if you have these details.